index.html

```diff
@@ -161,6 +163,31 @@ <h3 class="inner">skills</h3>
             </ul>
         </div>
     </div>
+</section>
+<section class="about-me">
+  <img src="images/Ellipse 3.svg" class="img1">
+  <img src="images/Rectangle 4.svg" class="img2">
+  <img src="images/Ellipse 12.svg" class="img3">
+  <img src="images/Ellipse 5.svg" class="img4">
+  <img src="images/Rectangle 7.svg" class="img5">
+  <img src="images/Shape.svg"class= "img6">
+<h1 class="cont">
+    Contact me
+  </h1>
+  <p class="contact-info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  <form action="#" class="form">
+    <label for="name" >
+      <input type="name" name="user_name"class="input" placeholder=" Enter your name" required >
```

Comment:
- [ ] Kindly make sure the name input has `maxlength="30"` attribute.
- [ ] Message input has `maxlength="500"` attribute.
- [ ] Kindly make sure all the field have the attribute `required`